unlock-app/src/hooks/useLocks.js

```diff
@@ -170,17 +168,21 @@ export const createLock = async (
  * A hook which yields locks
  * This hook yields the list of locks for the owner based on data from the graph and the chain
  * @param {*} address
+ * @param {*} network
  */
-export const useLocks = (owner) => {
-  const { network } = useAuth()
+export const useLocks = (owner, network) => {
```

Comment:
Please make sure it does not break anywhere else where we use `useLocks` ...

Reply:
already checked for `/dashboard` page, all works fine! but ill double check!